docs/getting-started/network-page-overview.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[network-page-overview]]
-= Network page overview
+= Network page
 
 The Network view provides key network activity metrics via an interactive map
 and provides network event tables that enable interaction with the Timeline. You
```

Comment:
Leaving this comment in the closest area possible since I can't actually comment on the lines I need to leave edits for. Looks like the tab names listed in lines 24-31 need to be bolded.